css/index.css

```diff
@@ -112,6 +135,37 @@ p {
     justify-content: center;
 }
 
```

Comment:
Great work using the parent styles to continue to style the children and only overriding the color for each box.